bot/utils/helpers.py

```diff
@@ -41,3 +47,19 @@ def remove_subdomain_from_url(url: str) -> str:
     netloc = extracted_url.registered_domain
     parsed_url = parsed_url._replace(netloc=netloc)
     return parsed_url.geturl()
+
+
+async def try_handle_forbidden(error: Forbidden, message: Message | None = None) -> bool:
+    """Attempts to handle `discord.Forbidden` errors, returning `True` if successful else `False`."""
+    if error.code == 90001:
+        # Occurs when the bot attempted to add a reaction
+        # to a message from a user that has blocked the bot.
+        log.debug("Failed to add reaction to a message since the message author has blocked the bot")
```

Comment:
```suggestion
        log.info(
            "Failed to add reaction to message %d-%d since the message author (%d) has blocked the bot",
            message.channel.id,
            message.id,
            message.author.id,
        )
```
Info to make it appear in logs by default, as I think this would be useful. Also including a reference to the message so we can easily figure out where it came from

Reply:
e4b2d5e